cohortextractor/patients.py

```diff
@@ -753,6 +755,60 @@ def registered_practice_as_of(
                     },
                 },
             )
+
+
+        ### Cluster RCTs
+
+        Clustered Randomised Control Trials (Cluster RCTs) that are clustered at practice level
+        are available. There is special syntax that signifies RCT. This is `rct__trialname__propertyname`
```

Comment:
Could add "A series of data files supplied by the trialists will be imported into OpenSafely; this will indicate which practices are enrolled, their assignment to an intervention group, and any other relevant practice properties or data gathered as part of the RCT outside of OpenSafely (e.g. number of GPs/nurses, number of practice visits made)."

Reply:
Also: "These RCT variables are only available for use by the researchers officially nominated by the responsible research group."